### PR TITLE
Server changes

### PR DESCRIPTION
**Pull request by javenkn**
-added angular controller and factory for getting the tweets
-server res.jsons the tweets
-page is displaying the emoji tweets
-deleted the public css file

### CODE REVIEW

**Review by adam-akamine — APPROVED** [review]
LGTM :+1:

File: twitter.js [hunk 1]
```diff
@@ -19,15 +18,22 @@ twitter.stream('statuses/filter', {'locations':'-180,-90,180,90'}, function (str
...
+              coordinates: coordinates,
+              date: date
+            }
+          );
+          // console.log(text);
           var unicode = text.map((emoji) => {
             return '\\u' + emoji.charCodeAt(0).toString(16) + '\\u' + emoji.charCodeAt(1).toString(16);
           });
-          console.log(unicode);
+          // console.log(unicode);
```
Comment (thgaskell) [inline twitter.js:32]: 🚽

File: twitter.js [hunk 2]
```diff
@@ -19,15 +18,22 @@ twitter.stream('statuses/filter', {'locations':'-180,-90,180,90'}, function (str
...
         }
         // console.log('Hello ' + String.fromCharCode(0xd83d, 0xde04)); //this is how to print an emoji based on the unicode
-        console.log(coordinates);
-        console.log(date);
+        // console.log(coordinates);
```
Comment (thgaskell) [inline twitter.js:35]: 🚽

File: twitter.js [hunk 3]
```diff
@@ -19,15 +18,22 @@ twitter.stream('statuses/filter', {'locations':'-180,-90,180,90'}, function (str
...
+        // console.log(date);
```
Comment (thgaskell) [inline twitter.js:36]: 🚽

**Review by thgaskell — COMMENTED** [review]
Please remove code instead of commenting it out.
It's a bad habit that will most likely lead to unnecessary merge conflicts in the future.

**Pull request merged**